### ISSUE TITLE
Parse data provided by SemEval-2013

### ISSUE THREAD

**Issue by bwbaugh** [bwbaugh/infer#9]
The [Sentiment Analysis in Twitter](http://www.cs.york.ac.uk/semeval-2013/task2/) task for SemEval 2013 provides training---and later testing---data for sentiment analysis. Since this data is hand-labeled (I believe by users on [Amazon Mechanical Turk](https://en.wikipedia.org/wiki/Amazon_Mechanical_Turk)), it makes for an attractive test set.

This relates to gh-8 because in order to add training data for an objective class we must first be able to parse the SemEval data files.

**Comment by bwbaugh** [comment]
I've completed this, but I'm trying to decide on whether or not I should commit the training data.

The reason for pause is that the training data has text from tweets downloaded from Twitter, which is either not allowed to share or okay to share depending on how you interpret the [Twitter API Terms of Use](https://dev.twitter.com/terms/api-terms). If I don't commit, then I'll need to change the unit tests since they rely on the training data being there (this might be a good idea regardless).
